conformance/tests/typeddicts_alt_syntax.py

```diff
@@ -42,4 +42,4 @@
 
 movie2: Movie2
 movie2 = {"name": "Blade Runner", "year": 1982}  # E?: May generate errors if keyword-argument syntax is unsupported
-movie2 = {"name": "Blade Runner", "year": ""}  # E: Incorrect type for year
```

Comment:
It feels like this error is also optional, because as it says above, the keyword argument syntax is not mandatory.

Reply:
Yes, that makes sense to me. If type checkers aren't expected to support this syntax for creating a TypedDict, they also don't need to understand it.